app/controllers/api/v1/user_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class UserController < ApiController
+      def update
+        if current_user.id == params[:id].to_i
+          current_user.update(user_params)
```

Comment:
Please always use the bang version
```suggestion
          current_user.update!(user_params)
```